### DISCUSSION TITLE
Are i18ns pulled from Transifex

### DISCUSSION THREAD

**Discussion by DragonZX** [iterate-ch/cyberduck#15086] [Ideas]
I found that some strings are translated in Transifex, but the latest pull was about 5 years ago, please guide me how can I help with that matter?

**Comment by dkocher** [comment]
We regularly pull changes from Transifex but have to migrate the job currently in Jenkins to a GitHub action instead.

**Comment by DragonZX** [comment]
Oh, I'm sorry, I haven't noticed that resource update for some strings was quite recent. However I found that we have some strings in the interface without translations (also not listed in transifex. Main menu and settings are affected. Could you please review it or guide me throwgh the code in order to make a patch?
![изображение](https://github.com/iterate-ch/cyberduck/assets/2978657/860afd39-a362-4b42-8580-28c2b8441bdc)

**Reply by dkocher** [reply]
You should find these in `Preferences.strings`. You can make contributions to the localization directly in Transifex.

**Reply by DragonZX** [reply]
@dkocher Hi, thank you for pulling changes, however I can't find in transifex these strings
Unlock Vault
Toggle Log Drawer
Open in Putty
Privacy Policy
Get a registration key!

Profiles

And also RRS hint 
![изображение](https://github.com/iterate-ch/cyberduck/assets/2978657/b3c06d91-ded7-4805-b3d0-45712099ab36)

Thank you for your work!

**Reply by dkocher** [reply]
* `Preferences.strings`

```
"Choose Reduced Redundancy Storage (RRS) to reduce costs by storing non-critical, reproducible data at lower levels of redundancy." = "Choose Reduced Redundancy Storage (RRS) to reduce costs by storing non-critical, reproducible data at lower levels of redundancy.";
````

* `Profiles.strings`

```
"Profiles" = "Profiles";
```


* `Cryptomator.strings`
```
"Unlock Vault" = "Unlock Vault";
```

**Reply by DragonZX** [reply]
I found strings you mentioned, but they were translated together with others, however are still untranslated at the interface.

**Reply by DragonZX** [reply]
Now I can see that that's the glitch for the windows specific strings, on MacOS looks great. On windows that string is untranslated
![изображение](https://github.com/iterate-ch/cyberduck/assets/2978657/d81993f6-da2f-4997-9262-00629c933c01)
But the opposite 
 
![изображение](https://github.com/iterate-ch/cyberduck/assets/2978657/8ef25fdd-81f9-436b-9d95-ec4e9beaff64)
Labels are not bringing to another line
